### ISSUE TITLE
validate incorrectly SKIPs label

### ISSUE THREAD

**Issue by rchenatjpl** [NASA-PDS-Incubator/validate#657]
### Checked for duplicates

No - I haven't checked

### 🐛 Describe the bug

For the attached bundle, validate SKIPped a file for a reason I don't see.
% validate -R pds4.bundle -t voy1
PDS Validate Tool Report
Configuration:
   Version                       3.3.0-SNAPSHOT
   Date                          2023-06-05T18:35:06Z
Parameters:
   Targets                       [file:/Users/rchen/Desktop/voy1/]
   Rule Type                     pds4.bundle
   Severity Level                WARNING
   Recurse Directories           true
   File Filters Used             [*.xml, *.XML]
   Data Content Validation       on
   Product Level Validation      on
   Allow Unlabeled Files         false
   Max Errors                    100000
   Registered Contexts File      /Users/rchen/PDS4tools/validate/resources/registered_context_products.json
Product Level Validation Results
  SKIP: file:/Users/rchen/Desktop/voy1/calib_freq/vgr_uso.xml
      INFO  [info.integrity.unreferenced_file]   Skipping file:/Users/rchen/Desktop/voy1/calib_freq/vgr_uso.xml due to collection not latest or does not sharing the same logical_identifier as the bundle target
  PASS: file:/Users/rchen/Desktop/voy1/bundle_voyager1_rss_jupiter_raw.xml
        1 product validation(s) completed
  PASS: file:/Users/rchen/Desktop/voy1/calib_freq/vgr_uso.xml
        2 product validation(s) completed
  PASS: file:/Users/rchen/Desktop/voy1/calib_freq/collection_calib_freq.xml
        3 product validation(s) completed
PDS4 Bundle Level Validation Results
  PASS: file:/Users/rchen/Desktop/voy1/calib_freq/collection_calib_freq.xml
        1 integrity check(s) completed
  PASS: file:/Users/rchen/Desktop/voy1/bundle_voyager1_rss_jupiter_raw.xml
        2 integrity check(s) completed
  PASS: file:/Users/rchen/Desktop/voy1/calib_freq/vgr_uso.xml
        3 integrity check(s) completed
Summary:
  0 error(s)
  0 warning(s)
  Product Validation Summary:
    3          product(s) passed
    0          product(s) failed
    1          product(s) skipped
  Referential Integrity Check Summary:
    3          check(s) passed
    0          check(s) failed
    0          check(s) skipped
End of Report
Completed execution in 2574 ms

### 🕵️ Expected behavior

I expected [...]

### 📜 To Reproduce

unzip the attached, then validate -R pds4.bundle -t voy1

### 🖥 Environment Info

- Version of this software [e.g. vX.Y.Z]
- Operating System: [e.g. MacOSX with Docker Desktop vX.Y]
...

### 📚 Version of Software Used

% validate -V
gov.nasa.pds:validate
Version 3.3.0-SNAPSHOT
Release Date: 2023-05-31 18:10:35

### 🩺 Test Data / Additional context

[voy1.zip](https://github.com/NASA-PDS/validate/files/11655664/voy1.zip)

### 🦄 Related requirements

🦄 #xyz

**Comment by al-niessner** [comment]
@jordanpadams @rchenatjpl 

No longer an error on the main branch:
```
$ validate -R pds4.bundle --skip-context-validation -t github657/bundle_voyager1_rss_jupiter_raw.xml
PDS Validate Tool Report

Configuration:
   Version                       3.3.0-SNAPSHOT
   Date                          2023-06-05T19:00:27Z

Parameters:
   Targets                       [file:/home/niessner/Projects/PDS/validate/src/test/resources/github657/bundle_voyager1_rss_jupiter_raw.xml]
   Rule Type                     pds4.bundle
   Severity Level                WARNING
   Recurse Directories           true
   File Filters Used             [*.xml, *.XML]
   Data Content Validation       on
   Product Level Validation      on
   Allow Unlabeled Files         false
   Max Errors                    100000
   Registered Contexts File      /home/niessner/Projects/PDS/validate/src/main/resources/util/registered_context_products.json



Product Level Validation Results

  PASS: file:/home/niessner/Projects/PDS/validate/src/test/resources/github657/bundle_voyager1_rss_jupiter_raw.xml
        1 product validation(s) completed

  PASS: file:/home/niessner/Projects/PDS/validate/src/test/resources/github657/calib_freq/vgr_uso.xml
        2 product validation(s) completed

  PASS: file:/home/niessner/Projects/PDS/validate/src/test/resources/github657/calib_freq/collection_calib_freq.xml
        3 product validation(s) completed

PDS4 Bundle Level Validation Results

  PASS: file:/home/niessner/Projects/PDS/validate/src/test/resources/github657/calib_freq/collection_calib_freq.xml
        1 integrity check(s) completed

  PASS: file:/home/niessner/Projects/PDS/validate/src/test/resources/github657/bundle_voyager1_rss_jupiter_raw.xml
        2 integrity check(s) completed

  PASS: file:/home/niessner/Projects/PDS/validate/src/test/resources/github657/calib_freq/vgr_uso.xml
        3 integrity check(s) completed

Summary:

  0 error(s)
  0 warning(s)

  Product Validation Summary:
    3          product(s) passed
    0          product(s) failed
    0          product(s) skipped

  Referential Integrity Check Summary:
    3          check(s) passed
    0          check(s) failed
    0          check(s) skipped


End of Report
Completed execution in 7259 ms
```

**Comment by al-niessner** [comment]
Ah, fixed in #652
